portfolio/src/main/webapp/index.html

```diff
@@ -104,6 +99,22 @@ <h4>CPP, JAVA, JAVASCRIPT, HTML, CSS, PYTHON</h4>
       <br>
     </div>
   </div>
+
+  <div id="contactInfos" class="tabcontent">
+
+  <form action="/form-handler" method="POST">
+    <p>Enter contact information here:</p>
+    <textarea name="text-input-name">Name</textarea>
```

Comment:
It is slightly annoying to clear the "Name" string before inputting the data I want. It would be better to use a placeholder tag. So something like
<textarea name="something" placeholder="Name"></textarea>